MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -455,72 +461,116 @@ private async Task<PlaybackInfoResponse> GetPlaybackInfo(Guid id, Guid userId, s
                 {
-                    if (!user.Policy.EnableAudioPlaybackTranscoding && !user.Policy.EnableVideoPlaybackTranscoding && !user.Policy.EnablePlaybackRemuxing)
+                    mediaSource.SupportsDirectStream = false;
+                }
+                else
+                {
+                    options.MaxBitrate = GetMaxBitrate(maxBitrate, user);
+                    
+                    if (item is Audio)
                     {
-                        options.ForceDirectStream = true;
+                        if (!user.Policy.EnableAudioPlaybackTranscoding)
+                        {
+                            options.ForceDirectStream = true;
+                        }
+                    }
+                    else if (item is Video)
+                    {
+                        if (!user.Policy.EnableAudioPlaybackTranscoding && !user.Policy.EnableVideoPlaybackTranscoding && !user.Policy.EnablePlaybackRemuxing)
+                        {
+                            options.ForceDirectStream = true;
+                        }
                     }
-                }
 
-                // The MediaSource supports direct stream, now test to see if the client supports it
-                var streamInfo = string.Equals(item.MediaType, MediaType.Audio, StringComparison.OrdinalIgnoreCase) ?
-                    streamBuilder.BuildAudioItem(options) :
-                    streamBuilder.BuildVideoItem(options);
+                    // The MediaSource supports direct stream, now test to see if the client supports it
+                    var streamInfo = string.Equals(item.MediaType, MediaType.Audio, StringComparison.OrdinalIgnoreCase) ?
+                        streamBuilder.BuildAudioItem(options) :
+                        streamBuilder.BuildVideoItem(options);
 
-                if (streamInfo == null || !streamInfo.IsDirectStream)
-                {
-                    mediaSource.SupportsDirectStream = false;
-                }
+                    if (streamInfo == null || !streamInfo.IsDirectStream)
+                    {
+                        mediaSource.SupportsDirectStream = false;
+                    }
 
-                if (streamInfo != null)
-                {
-                    SetDeviceSpecificSubtitleInfo(streamInfo, mediaSource, auth.Token);
-                }
+                    if (streamInfo != null)
+                    {
+                        SetDeviceSpecificSubtitleInfo(streamInfo, mediaSource, auth.Token);
+                    }
+				}
             }
 
             if (mediaSource.SupportsTranscoding)
             {
-                options.MaxBitrate = GetMaxBitrate(maxBitrate, user);
-
+				if (mediaSource.IsRemote && user.Policy.ForceRemoteSourceTranscoding)
+				{
+					if (GetMaxBitrate(maxBitrate, user) < mediaSource.Bitrate)
+					{
+						options.MaxBitrate = GetMaxBitrate(maxBitrate, user);
+					}
+					else
+					{
+						options.MaxBitrate = mediaSource.Bitrate;
```

Comment:
why? original bitrate does not directly correlate with transcoded one

Reply:
Yeah, looking at it now. If the remote stream is only slightly lower than the max bitrate allowed, this could allow the stream to exceed the max bitrate allowed in the user policy when it becomes transcoded.

Reply:
So testing bandwidth restrictions, it behaves as expected. With that said, I think I can remove the change and have the same behavior.

Reply:
MaxBitrate is the maximum bitrate that the user or admin allows and doesn't seem related to what the maximum bitrate of the media is. Based on my understanding, I don't agree with your change.